code/src/App.js

```diff
@@ -1,9 +1,47 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { compose, createStore, combineReducers } from "@reduxjs/toolkit";
+import persistState from "redux-localstorage";
+import { projects } from "reducers/projects";
+import { colors } from "reducers/colors";
+import { icons } from "reducers/icons";
+import Home from "components/Home";
+import Error from "components/Error";
+import Project from "components/Project";
+import Header from "components/Header";
+import NewProject from "components/NewProject";
+import ErrorBoundary from "components/ErrorBoundary";
+
+const enhancer = compose(persistState());
+
+const reducer = combineReducers({
```

Comment:
We have previously worked with one reducer before but three is new! Way to go to adding more to your project :)